addons/binding/org.openhab.binding.sleepiq/src/main/java/org/openhab/binding/sleepiq/SleepIQBindingConstants.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sleepiq;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link SleepIQBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Gregory Moyer - Initial contribution
+ */
+public class SleepIQBindingConstants {
+
+    public static final String BINDING_ID = "sleepiq";
+
+    // List of all Thing Type UIDs
+
+    // bridge
+    public static final ThingTypeUID THING_TYPE_CLOUD = new ThingTypeUID(BINDING_ID, "cloud");
+
+    // generic thing types
+    // public static final ThingTypeUID THING_TYPE_SINGLE_BED = new ThingTypeUID(BINDING_ID, "singleBed");
+    public static final ThingTypeUID THING_TYPE_DUAL_BED = new ThingTypeUID(BINDING_ID, "dualBed");
+
+    // List of all Channel ids
+    // public static final String CHANNEL_BED_IN_BED = "bed#inBed";
+    public static final String CHANNEL_LEFT_IN_BED = "left#inBed";
```

Comment:
I don't quite understand the comment here. What do you mean by "dashed in the channel name"? The '#' character is used here because I created channel groups (one for the left chamber and another for the right). Can you elaborate a bit more?